masterlist.yaml

```
@@ -2394,6 +2394,52 @@ plugins:
       - crc: 0xA6F1A3B3
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
 
+  - name: '(JS Dragon Claws AE - .*|JS_DBM_(HelgenReborn|KA|SkyrimSewers|Wyrmstooth)_Patch)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/57038/'
+        name: 'JS Dragon Claws AE (Anniversary Edition)'
+  - name: '(JS Dragon Claws AE - (Helgen Reborn|Konahriks Accoutrements|Legacy of the Dragonborn|Skyrim Sewers|Wyrmstooth)|JS_DBM_SkyrimSewers_Patch)\.esp'
+    tag: [ Graphics ]
+  - name: '(JS Dragon Claws AE - (Helgen Reborn|Konahriks Accoutrements|Skyrim Sewers|Wyrmstooth))\.esp'
```

There's an unnecessary capture group in this regex:

```suggestion
  - name: 'JS Dragon Claws AE - (Helgen Reborn|Konahriks Accoutrements|Skyrim Sewers|Wyrmstooth)\.esp'
```